corehq/motech/repeaters/migration_functions.py

```diff
@@ -0,0 +1,27 @@
+"""
+Functions used by RunPython operations in migrations.
+
+.. note::
+   Placing them in a separate module makes it easier to import them in
+   unittests.
+
+"""
+from django.db import transaction, IntegrityError
+
+from corehq.motech.repeaters.dbaccessors import iter_repeaters
+from corehq.motech.repeaters.models import RepeaterStub
+
+
+def create_repeaterstubs(apps, schema_editor):
+    for repeater in iter_repeaters():
+        with transaction.atomic():
```

Comment:
Any thought given to doing this process via [bulk_create](https://docs.djangoproject.com/en/3.2/ref/models/querysets/#bulk-create)? It should be substantially faster